lib/IDE/CompletionLookup.cpp

```diff
@@ -1679,7 +1679,20 @@ void CompletionLookup::addNominalTypeRef(const NominalTypeDecl *NTD,
   addLeadingDot(Builder);
   Builder.addBaseName(NTD->getName().str());
 
-  addTypeAnnotation(Builder, NTD->getDeclaredType());
+  // "Fake" annotation for custom attribute types.
+  StringRef customAttributeAnnotation;
+  if (NTD->getAttrs().hasAttribute<PropertyWrapperAttr>()) {
+    customAttributeAnnotation = "Property Wrapper";
+  } else if (NTD->getAttrs().hasAttribute<ResultBuilderAttr>()) {
+    customAttributeAnnotation = "Result Builder";
+  } else if (NTD->isGlobalActor()) {
+    customAttributeAnnotation = "Global Actor";
+  }
```

Comment:
Should we support multiple? E.g. `@propertyWrapper @resultBuilder struct MyStruct { .. }`